bigtable/google/cloud/bigtable/row_data.py

```diff
@@ -405,10 +429,7 @@ def _read_next(self):
 
     def _read_next_response(self):
         """Helper for :meth:`__iter__`."""
-        retry_ = retry.Retry(
-            predicate=_retry_read_rows_exception,
-            deadline=60)
-        return retry_(self._read_next, on_error=self._on_error)()
+        return self.retry(self._read_next, on_error=self._on_error)()
```

Comment:
We need an assertion somewhere or a new test that shows `self.retry` being used here.

Reply:
Added a new test "test_yield_retry_rows_data".

Reply:
@sumit-ql Did you push that change?  I don't see any new code.